src/Microsoft.AspNet.Mvc.Core/ConsumesAttribute.cs

```diff
@@ -53,13 +64,12 @@ public void OnResourceExecuting(ResourceExecutingContext context)
             // Ignore all other filters. This is to ensure we have a overriding behavior.
             if (IsApplicable(context.ActionDescriptor))
             {
-                MediaTypeHeaderValue requestContentType = null;
-                MediaTypeHeaderValue.TryParse(context.HttpContext.Request.ContentType, out requestContentType);
+                var requestContentType = context.HttpContext.Request.ContentType;
 
                 // Confirm the request's content type is more specific than a media type this action supports e.g. OK
                 // if client sent "text/plain" data and this action supports "text/*".
                 if (requestContentType != null &&
```

Comment:
No, in fact, I could remove requestContentType != null as MediaTypeComparisons.IsSubsetOf will do the right thing. The check for null is just a shortcut to not even try in that situation

Also, requestContentType is not a property that a user will normally set (they would need to be doing some extensibility work to do so) so I don't think is worth the change